.golangci.yml

```diff
@@ -40,6 +40,9 @@ issues:
     - path: schemas/.*
       linters:
         - misspell
+    - linters:
+        - staticcheck
+      text: "SA1019:"
```

Comment:
what does this change do?

Reply:
we skip the lint check for old package: `commands/daemon.go:6:2: SA1019: package io/ioutil is deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)`

[old lint logs](https://app.circleci.com/pipelines/github/filecoin-project/lily/4109/workflows/455c81c2-a098-4524-9676-abef485dd8d3/jobs/20185).

Reply:
Instead of silencing the linter errors, replace the deprecated dependency: `io/ioutil` with either the `io` or `os` package.

Reply:
Fixed!